### PR TITLE
Features/remove custom dir

### PR DESCRIPTION
**Pull request by ghost**
Load from /packages and format the code with .editorconfig

### CODE REVIEW

File: lib/core_modules/module/index.js [hunk 1]
```diff
@@ -68,17 +68,15 @@ function onModulesEnabled(meanioinstance, defer) {
...
+    q.all(defers).done(onModulesEnabled.bind(null, meanioinstance, defer));
 }
 //
 
 function findModules(meanioinstance, defer, app) {
     var disabled = _.toArray(meanioinstance.getConfig().disabledModules);
-    Q.all([
+    q.all([
         search(_modules, disabled, 'packages'),
-        search(_modules, disabled, 'packages/core'),
```
Comment (ghost) [inline lib/core_modules/module/index.js:79]: @rjVapes here is the loading that I modified

Comment (ADSKLowenthal) [reply lib/core_modules/module/index.js:79]: LGTM

File: lib/core_modules/module/index.js [hunk 2]
```diff
@@ -223,4 +221,4 @@ function supportModules(Meanio) {
 
 }
 
-module.exports = supportModules;
+module.exports = supportModules;
```
Comment (ADSKLowenthal) [inline lib/core_modules/module/index.js]: Did the final new line in editorconfig not catch this?

File: lib/core_modules/module/dependablelist.js [hunk 3]
```diff
@@ -1,58 +1,60 @@
+'use strict';
```
Comment (ADSKLowenthal) [inline lib/core_modules/module/dependablelist.js:1]: Nothing to change now, but is this no longer necessary if we require Node 6? Is there an es6 improvement that makes this unnecessary going forward?

File: lib/mean.js [hunk 4]
```diff
@@ -13,18 +13,18 @@ function Meanio() {
   if (this.active) return;
   Meanio.Singleton = this;
   this.version = require('../package').version;
-  this.instanceWaitersQ = [];
+  this.instanceWaitersq = [];
```
Comment (ADSKLowenthal) [inline lib/mean.js]: Should probably still be a capital Q here

File: lib/core_modules/module/index.js (hunk 2 above)
Comment (ghost) [reply lib/core_modules/module/index.js]: odd, i don't know what happened there.  Will fix

**Comment by clowenhg** [comment]
@rjVapes @vtdave Do we want to merge these into our master? Or should we be PRing them against linnovate/master and then just updating the master here?

**Pull request closed without merging**